frontend/rolecall/src/app/performance/performance-editor.component.ts

```diff
@@ -154,7 +154,9 @@ export class PerformanceEditor implements OnInit, OnDestroy, AfterViewChecked {
     this.state.status = PerformanceStatus.DRAFT;
     // Saving casts results in an immediate cast load.
     // Clear a key parameter to prevent deadly embrace
-    this.castDnD.castSelected = false;
+    if (this.castDnD) {
```

Comment:
This is fine, but can we also fix the type of `castDnD` while you make this change? It could potentially highlight more related bugs.

`@ViewChild('castDnD') castDnD?: CastDragAndDrop;`